static/js/protein/chart.ts

```diff
@@ -219,6 +240,76 @@ function addYLabel(
     .text(labelText);
 }
 
+export function drawTissueLegend(id: string, data: ProteinStrData[]): void {
+  // checks if the data is empty or not
+  if (_.isEmpty(data)) {
+    return;
+  }
+  const svg = d3
+    .select("#tissue-score-legend")
+    .append("svg")
+    .attr("width", GRAPH_WIDTH_XL)
+    .attr("height", GRAPH_HEIGHT_XS);
+  const data0 = d3.keys(TISSUE_LEGEND_DICT);
+  // slicing the dictionary in half to display the legend in two rows
+  const data1 = data0.slice(0, 8);
+  const data2 = data0.slice(8, 15);
+  // creating circles for the first row in legend
+  svg
+    .selectAll("mydots1")
+    .data(data1)
+    .enter()
+    .append("circle")
+    .attr("cx", (d, i) => {
+      return NUM_DATA_POINTS + i * GRAPH_HEIGHT_XS;
+    })
+    .attr("cy", 50)
+    .attr("r", 7)
+    .style("fill", (d) => {
+      return TISSUE_LEGEND_DICT[d];
+    });
+  svg
+    .selectAll("legend-text1")
```

Comment:
since the text on the first row and the second row should be styled the same way, don't need to distinguish between "legend-text1" and "legend-text2", can just be "legend-text"

Reply:
Done!